### PR TITLE
Connectify user-facing output

### PR DESCRIPTION
**Pull request by cocreature**
Together with #7615, this overs all items in #7612.

changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by garyverhaegen-da — APPROVED** [review]

File: bazel_tools/pom_template.xml [hunk 1]
```diff
@@ -20,15 +20,6 @@
...
   </licenses>
 
-  <developers>
-    <developer>
-      <name>Digital Asset SDK Feedback</name>
-      <email>sdk-feedback@digitalasset.com</email>
-      <organization>Digital Asset (Switzerland) GmbH</organization>
-      <organizationUrl>http://www.daml.com</organizationUrl>
-    </developer>
-  </developers>
```
Comment (hurryabit) [inline bazel_tools/pom_template.xml:30]: Don't we want a reference to DAML Connect here now?

**Review by hurryabit — COMMENTED** [review]
Thank you so much for pushing through this.

File: bazel_tools/pom_template.xml (hunk 1 above)
Comment (cocreature) [reply bazel_tools/pom_template.xml:30]: @bame-da suggested to kill it completely and I agree that the pom file in our example sis not the right place for support.

Comment (hurryabit) [reply bazel_tools/pom_template.xml:30]: That makes sense. Thank you.

**Pull request merged**